### PR TITLE
Fix for active Accordion not starting opened

### PR DESCRIPTION
**Pull request by marlonpp**
#### What does this PR do?
Fix for active Accordion not starting opened

#### Where should the reviewer start?
components/Accordion.js

#### What testing has been done on this PR?
Started some basic Accordion tests

#### How should this be manually tested?
Create an Accordion with tag active

#### Do the grommet docs need to be updated?
Yes, will submit another PR

### CODE REVIEW

File: src/js/components/Accordion.js [hunk 1]
```diff
@@ -1,10 +1,10 @@
 // (C) Copyright 2014-2016 Hewlett Packard Enterprise Development LP
 
-import React, { Component, PropTypes } from 'react';
```
Comment (alansouzati) [inline src/js/components/Accordion.js]: Hey, I believe there is something going on with the re-ordering of the imports. I'm assuming your IDE is sorting them as you save the file?

Can we revert this change?

File: src/js/components/Accordion.js [hunk 2]
```diff
@@ -39,7 +39,7 @@ export default class Accordion extends Component {
     const accordionChildren = React.Children.map(children, (child, index) => {
       return React.cloneElement(child, {
         id: 'accordion-panel-' + index,
-        active: !openMulti ? (this.state.activeIndex === index) : null,
+        active: !openMulti ? (this.state.activeIndex === index && child.props.active) : child.props.active,
```
Comment (alansouzati) [inline src/js/components/Accordion.js]: I believe the right check should be:

`active: !openMulti ? (this.state.activeIndex === index) : child.props.active,`

File: src/js/components/Accordion.js [hunk 3]
```diff
@@ -67,5 +67,6 @@ Accordion.propTypes = {
 
 Accordion.defaultProps = {
   openMulti: false,
-  animate: true
+  animate: true,
+  initialIndex: 0
```
Comment (alansouzati) [inline src/js/components/Accordion.js]: I believe we don't need an initialIndex

File: src/js/components/Collapsible.js [hunk 4]
```diff
@@ -45,19 +45,21 @@ class Collapse extends Component {
...
     );
-
     return <div {...this.props} className={classes} />;
   }
 };
 
 class Collapsible extends Component {
   render () {
     const Component = this.props.animate ? TransitionGroup : Box;
+    let collapseProps = Object.assign({}, this.props);
```
Comment (alansouzati) [inline src/js/components/Collapsible.js]: for some reason, Object.assign is messing up with the animation. When I'm testing this the animation gets laggy.

Can we replace this by the following? 

`const collapseProps = Props.omit(this.props, Object.keys(Collapsible.propTypes));`

Props is available at:

`import Props from '../utils/Props';`

File: test/components/Accordion-test.js [hunk 5]
```diff
@@ -0,0 +1,102 @@
+// (C) Copyright 2014-2015 Hewlett-Packard Development Company, L.P.
```
Comment (alansouzati) [inline test/components/Accordion-test.js]: Way to go, thanks for writing the test 👍

File: src/js/components/Accordion.js (hunk 3 above)
Comment (ericsoderberghp) [inline src/js/components/Accordion.js]: Agreed. And, we should align Tabs with Accordion in this respect as well.

File: src/js/components/Collapsible.js (hunk 4 above)
Comment (ericsoderberghp) [inline src/js/components/Collapsible.js]: Manny had a nice ES6 pattern for this where he enumerated all props and then used a ellipsis to capture the rest of them, as in `const { prop1, prop2, ...otherProps }`.

**Comment by alansouzati** [comment]
Nice man! Thanks for your contribution.
